components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/OAuth2ServiceTest.java

```diff
@@ -71,10 +102,20 @@
     AuthenticatedUser authenticatedUser;
 
     @Mock
+    OAuthEventInterceptor oAuthEventInterceptorProxy;
```

Comment:
shall we make this variables as private?

Reply:
okay